ui/react-components/components/DatePicker/DatePicker.jsx

```diff
@@ -15,17 +15,18 @@ import DatePickerCustomHeader from './DatePickerCustomHeader.jsx';
 const datetoMomentDate = (date) => date==='' || date===null || date === undefined ? date
     : moment(date,'MM/DD/YYYY' );
 const AppointmentDatePicker = (props) => {
-    const {minDate, value, onChange, isDisabled} = props;
+    const {minDate, value, onChange, isDisabled, hide} = props;
     let calendarStyles =[appointmentDatePicker];
     isDisabled && calendarStyles.push(disable);
     const selectedDate = value?  value.toDate() : value;
 
     let containerStyles=[];
     isDisabled && containerStyles.push(disable);
 
-    const handleBlurOfDateInput = (date) =>{
-        const valueEntered= datetoMomentDate(date);
-        onChange(valueEntered)
+    const handleBlurOfDateInput = (date) => {
+        const valueEntered = datetoMomentDate(date);
+        hide && hide();
```

Comment:
what happens if i focus on date input and then click on calendar date?

Reply:
Fixed